docker-compose.yml

```diff
@@ -25,3 +26,11 @@ services:
     environment:
       POSTGRES_PASSWORD: main
       POSTGRES_USER: main
+    ports:
+      - 5432:5432
+    volumes:
+      - pyslackersweb-postgresql-data:/var/lib/postgresql/data
+
+volumes:
+  pyslackersweb-postgresql-data:
+    name: pyslackersweb-postgresql-data
```

Comment:
It looks to me that `name` is just noise and can be removed: https://docs.docker.com/compose/compose-file/#name. It's valid to have (removing the unnecessary namespace in the name):

```yaml
volumes:
  pgdata:

# or

volumes:
  pgdata: {}
```